docs/docs/50-user-guide/60-reference-docs/30-promotion-steps/snow-create.md

```diff
@@ -0,0 +1,379 @@
+---
+
+# `snow-create`
+
+<span class="tag professional"></span>
+<span class="tag beta"></span>
+
+:::info
+This promotion step is only available in Kargo on the
+[Akuity Platform](https://akuity.io/akuity-platform), versions v1.9.0 and above.
+:::
+
+This page is for `snow-create` step. You can create a ServiceNow record using this step. Check below for other ServiceNow promotion steps.
+
+ServiceNow integration for kargo is a group of promotion steps:  
+1. [snow-create](./snow-create.md) 
+2. [snow-update](./snow-update.md)
+3. [snow-delete](./snow-delete.md)
+4. [snow-query-for-records](./snow-query-for-records.md)
+5. [snow-wait-for-condition](./snow-wait-for-condition.md)
+
+These steps provide a comprehensive integration with ServiceNow, allowing you to create, update, delete, query for records, and track
+promotion workflows. This is particularly useful for maintaining traceability between
+your promotion processes and project management activities.
+
+All the steps above support different record types in ServiceNow including managing Change Requests, Incidents, 
+Problems etc.,. This integration also provides condition tracking through `snow-wait-for-condition`, making it a powerful tool 
+for promotion workflows that require coordination with project management systems.
+
+`snow-create` step supports two APIs:  
+#### 1. Change Management API
+This API is used primarily for managing Change Requests.
+
+Official documentation is available [here](https://www.servicenow.com/docs/bundle/zurich-api-reference/page/integrate/inbound-rest/concept/change-management-api.html).
+
+URL format: `/api/sn_chg_rest/{api_version}/change/{change_sys_id}/task/{task_sys_id}`
+
+#### 2. Table API
+This API is used for managing Incidents, Problems, and other record types.
+
+Official documentation is available [here](https://www.servicenow.com/docs/bundle/zurich-api-reference/page/integrate/inbound-rest/concept/c_TableAPI.html).
+
+URL format: `/api/now/{api_version}/table/{tableName}/{sys_id}`
+
+### Using the API
+
+Most of the time you cannot use the field labels you see in the ServiceNow UI as keys in the REST API. 
+For example, if you want to set the value for the “Short description” field:
+
+![](./images/snow-short-des.png)
+
+You can't use `Short description` in the REST API. You need to use `short_description` as the key and set the value via REST API parameters.
+To find the correct key for a field, right-click on the field and click `Configure Dictionary`:
+
+![](./images/snow-configure-dict.png)
+
+`column_name` is the key:
+
+![](./images/snow-col-name.png)
+
+To see choices available for a particular field e.g., `State`, right click on the field and click on `Show Choice List` to see the available choices for use in the steps:
+
+![](./images/show-choices.png)
+
+For example `New` is `-1` and `Scheduled` is `-2`
+
+![](./images/choice-list.png)
+
+## Configuration
+
+### Credentials
```

Comment:
As for the headings please follow the format we use in other steps, Configuration is used instead of the Record heading adn so on.